app/src/main/java/com/sopt/instagram/data/repository/GetFriendStoriesRepositoryImpl.kt

```diff
@@ -0,0 +1,17 @@
+package com.sopt.instagram.data.repository
+
+import com.sopt.instagram.data.source.GetFriendStoriesSource
+import com.sopt.instagram.domain.entity.GetFriendStoriesEntity
+import com.sopt.instagram.domain.repository.GetFriendStoriesRepository
+import javax.inject.Inject
+
+class GetFriendStoriesRepositoryImpl @Inject constructor(
+    private val getFriendStoriesSource: GetFriendStoriesSource,
+) : GetFriendStoriesRepository {
+    override suspend fun getFriendStories(): Result<List<GetFriendStoriesEntity>?> =
+        runCatching {
+            getFriendStoriesSource.getFriendStoriesList().data?.map {
+                it.toGetFriendStoriesEntity()
+            }
+        }
```

Comment:
```suggestion
        runCatching {
            getFriendStoriesSource.getFriendStoriesList().data?.map { response ->
                response.toFriendStory()
            }
        }
```

스코프 함수에서 `it` 보다는 명시적인 변수명을 사용하면 좋을 것 같습니다!
변수 네이밍은 마음대로 해주세요!

Reply:
넵!!!